### DISCUSSION TITLE
Is there a way we can customize the buckets in System.Diagostics.Metrics.Histogram

### DISCUSSION THREAD

**Discussion by adugar-conga** [dotnet/runtime#72765] [Q&A]
Is there a way we can customize the buckets in System.Diagostics.Metrics.Histogram in .Net library ?

Currently the buckets are in the range of 

(0,5]:0
(0,10]:0
(0,25]:0
(0,50]:0
(0,75]:0
(0,100]:0
(0,250]:1
(0,500]:0
(0,1000]:0
(0,+Infinity]:0

Is there a way we can customize the above buckets in histogram metrics ?

**Comment by itstudes** [comment]
Consider looking at these links:
- https://github.com/open-telemetry/opentelemetry-dotnet/discussions/3245
- https://medium.com/@thepratikswami/how-to-add-custom-buckets-in-histogram-metrics-2c46675e6c32
